packages/scene-composer/src/layouts/SceneLayout/components/ScenePanel.tsx

```diff
@@ -8,15 +8,16 @@ import './ScenePanel.scss';
 interface PanelType {
   direction: typeof Direction.Left | typeof Direction.Right;
   panels: Record<string, JSX.Element>;
+  isViewing?: boolean;
```

Comment:
No need to pass `isViewing` as a prop. This can be determined using useStore hook.

`const sceneComposerId = useSceneComposerId();`
` const isViewing = useStore(sceneComposerId)((state) => state.isViewing());`

Reply:
It's a good suggestion, however, the value for `isViewing` coming from `useStore` gets modified on multiple re-renders & doesn't behave as you would expect. The prop being passed works as intended & does not create any additional dependency that didn't already exist. This shouldn't be a blocker.

Reply:
Per sync with @mukeshsahay we've modified the prop from `isViewing` to `collapse` to improve readability & maintain the flexibility of the component. :D